projects/tinkerbell/hub/docker/linux/reboot/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+ARG BUILDER_IMAGE
+
+FROM $BUILDER_IMAGE as touch-builder
+
+RUN set -x && \
+    yumdownloader --destdir=/tmp/downloads coreutils && \
+    cd /newroot && \
+    rpm2cpio /tmp/downloads/coreutils*.rpm | cpio -idv ./usr/bin/touch
```

Comment:
is touch a statically compiled binary?  if you run `ldd touch` does it show any dynamic libs?

Reply:
```
#7 7.768 + ldd ./usr/bin/touch
#7 7.772        linux-vdso.so.1 (0x00007ffe41b4f000)
#7 7.772        libc.so.6 => /lib64/libc.so.6 (0x00007f7f25c90000)
#7 7.772        /lib64/ld-linux-x86-64.so.2 (0x00007f7f2603b000)
#7 DONE 8.4s
```

Reply:
Do we need flexibility over the base image? I ask because fiddling with libc versions can result in unrunnable binaries and letting maintainers twiddle that without knowing intricacies can result in confusion and a lot of time burnt.

I guess I'm wondering if its better to house configuration in a Makefile or, given the singular use of this image, just specify the FROM images as constants so there's no room for error.